api-server-nest/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "infinite-api",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
```

Comment:
Should probably change this to `"MIT"` in case we end up copying this directory to `api-server` like we did with the Nuxt refactor.